acai-ios/MenuViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MenuViewController.swift
+//  acai-ios
+//
+//  Created by Jaewon Sim on 3/13/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+import SnapKit
+
+class MenuViewController: UIViewController {
+    
+    var collectionView: UICollectionView!
+    var adapter: ListAdapter!
```

Comment:
Could we call this listAdapter to be consistent with our other view controllers.